GitHubRepositoryMerge.ps1

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+function Merge-GitHubRepositoryBranches
```

Comment:
This should have failed a `PSScriptAnalyzer` check for naming a function using the plural syntax.  Please consider renaming to `Merge-GitHubRespositoryBranch`